scripts/pi-hole/js/db_queries.js

```diff
@@ -324,8 +329,8 @@ $(function () {
         }
       },
       { width: "10%" },
-      { width: "40%" },
-      { width: "20%", type: "ip-address" },
+      { width: "40%", render: $.fn.dataTable.render.text() },
```

Comment:
Why this extra step? Wouldn't it work without this and showing what was really queried (even when not executed because `&lt;`,etc. )? I don't think forcing `&lt;` in this field rather than `<` is the way to go, but you may have reasons to do so.

Reply:
for consistency with `queries.js` (also, without this it will still execute)

Reply:
So does it show `<` or does it show `&lt;` ?